test/unit/reporting/claim-reporting-fees-no-fork.js

```diff
@@ -0,0 +1,552 @@
+/* eslint-env mocha */
+
+/**
+ * @todo Create shared unit test files to reduce amount of redundant code for testing claimReportingFees.
+ */
+
+"use strict";
+
+var assert = require("chai").assert;
+var assign = require("lodash.assign");
+var BigNumber = require("bignumber.js");
+var proxyquire = require("proxyquire").noPreserveCache();
+var noop = require("../../../src/utils/noop");
+var sinon = require("sinon");
+
+describe.only("reporting/claim-reporting-fees", function () {
```

Comment:
Probably want to remove .only

Reply:
Ah, yeah. Thanks, I forgot to remove that before committing.